conf/drupal/config/geocoder.settings.yml

```diff
@@ -1,47 +1,4 @@
 cache: true
 _core:
   default_config_hash: 51p0N1fcuoTUrOlPIReSbz6lCjo-zJRgB5VSq7FGk3Y
-plugins_options:
-  arcgisonline:
-    sourcecountry: ''
-    usessl: false
-  bingmaps:
-    apikey: ''
-    locale: ''
-  geonames:
-    username: ''
-    locale: ''
-  googlemaps:
-    apikey: ''
-    locale: ''
-    region: ''
-    usessl: true
-  ipinfodb:
-    apikey: ''
-    precision: city
-  mapquest:
-    apikey: ''
-    licensed: false
-  maxmind:
-    apikey: ''
-    service: f
-    usessl: false
-  nominatim:
-    rooturl: ''
-    locale: ''
-  opencage:
```

Comment:
@weitzman Opencage was a configured plugin before the upgrade, and we have some settings in settings.php. It looks like at some point we switched to this to get around Google's daily limit ([link](https://jira.mass.gov/browse/DP-5989)). Any idea if this is still being used or if we can remove configuration for it?

Reply:
I don't know anything about that sorry. I verified that we are storing and showing address correctly so some plugin must be doing that.